dandi/tests/test_files.py

```diff
@@ -305,7 +306,11 @@ def test_dandi_file_zarr_with_excluded_dotfiles(tmp_path: Path) -> None:
     )
     with pytest.raises(UnknownAssetError):
         dandi_file(zarr_path)
-    (zarr_path / "arr_0" / "foo").touch()
+    with (zarr_path / "arr_0" / "foo").open("w") as fp:
+        print("Text.", file=fp)
+        # Force changes to be synced when testing on NFS:
+        fp.flush()
+        os.fsync(fp.fileno())
```

Comment:
just curious -- `.write_text()` isn't sufficient enough? i.e. that any regular file writing without explicit `fsync` is no good?

Reply:
I tried `write_text()`, and the test failed on NFS with the same error as before.